From 08f95083b9b772acd7b516b62d81ee17c1c6d4c5 Mon Sep 17 00:00:00 2001
From: Yang Wei <Wei.Yang@windriver.com>
Date: Fri, 9 Aug 2013 11:17:06 +0800
Subject: [PATCH 428/430] gianfar: remove unnecessary code

In gfar_probe, we do not need to disable device wakeup as
the wakeup source only is not registered while enabling device
wake up.

Signed-off-by: Yang Wei <Wei.Yang@windriver.com>
---
 drivers/net/ethernet/freescale/gianfar.c |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/freescale/gianfar.c b/drivers/net/ethernet/freescale/gianfar.c
index d899b0c..333dec8 100644
--- a/drivers/net/ethernet/freescale/gianfar.c
+++ b/drivers/net/ethernet/freescale/gianfar.c
@@ -1494,10 +1494,8 @@ static int gfar_probe(struct platform_device *ofdev)
 		goto register_fail;
 	}
 
-	if (priv->wol_supported) {
+	if (priv->wol_supported)
 		device_set_wakeup_capable(&ofdev->dev, true);
-		device_set_wakeup_enable(&ofdev->dev, false);
-	}
 
 	/* fill out IRQ number and name fields */
 	for (i = 0; i < priv->num_grps; i++) {
-- 
1.7.5.4

